src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer starts work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stops work");

```

Comment:
Similarly, the message 'Bulldozer stops work' should be more informative and indicate that it's a Bulldozer machine that has stopped working.